Composer/packages/server/src/models/bot/feedManager.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as msRest from '@azure/ms-rest-js';
+
+import logger from './../../logger';
+
+const log = logger.extend('feed-manager');
+
+// http client for fetch json feed from url
+const clientOptions: msRest.ServiceClientOptions = {
+  requestPolicyFactories: [msRest.logPolicy(log)],
+};
+const token = process.env.ACCESS_TOKEN || 'token';
+const creds = new msRest.TokenCredentials(token);
+const client = new msRest.ServiceClient(creds, clientOptions);
+
+// TODO: update feed url to main
+const feedUrl = 'https://raw.githubusercontent.com/microsoft/BotFramework-Composer/zhixzhan/homepage-refine/feeds.json';
```

Comment:
@zhixzhan Do we want this JSON on the Composer repo directly? Seems like if the homepage is driven off this JSON we would need it on a CDN. In the same way have you thought about versioning this JSON file. Basically, going forward we might want to pull the content based on the Composer version that is currently running to get appropriate content.

@boydc2014

Reply:
suggestion 1: hard code aka.ms link instead of raw links

Reply:
> What happens if the feed URL gives a 404 or an error status code. We can still create a new bot etc correct?

Yes, we can, the functionality will not be effect by those feed widgets.